setup.cfg

```diff
@@ -26,6 +26,7 @@ python_requires = >=3.7
 zip_safe = False
 install_requires =
     invenio-records[postgresql]>=2.0.1
+    sqlalchemy-utils[encrypted]>=0.38.3
```

Comment:
sqlalchemy-utils would come in by package inheritance from invenio-records, but it would not bring the deps for the encrypted extra